src/front/js/component/Testimonials.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import user1 from "../../img/paty.jpg";
+import user2 from "../../img/nan.jpg";
+import "../../styles/Testimonials.css";
+
+const Home = () => {
+  return (
+    <React.Fragment>
+      <section className="content-container">
+        <div className="textArea">
```

Comment:
I'd rather use snake_case (or a variant, since we use dashes instead of underscores) for class names like these. Besides that, textArea is a tag name, so that might get confusing as well...

I'd leave it as "text-area", instead.

Reply:
is ok